sdk/eventhubs/azure-messaging-eventhubs/src/eventhubs_utilities.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "private/eventhubs_utilities.hpp"
+
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+
+namespace Azure { namespace Messaging { namespace EventHubs { namespace _detail {
+  namespace {
+
+    size_t LogRawData(std::ostream& os, size_t startOffset, const uint8_t* const pb, size_t cb)
+    {
+      // scratch buffer which will hold the data being logged.
+      std::stringstream ss;
+
+      size_t bytesToWrite = (cb < 0x10 ? cb : 0x10);
```

Comment:
What's the significance of 0x10? Worth a comment or named constant.

Reply:
Also, what are pb, and cb?

Reply:
0x10 is hex 10, or 16 bytes.

pb is pointer to a byte
cb is a count of bytes.

I'll make it clearer. Basically this is a chunk of code I've used for years to dump binary blobs to a logger.